### PR TITLE
Replace sed -r with awk so it works on macOS too

### PR DESCRIPTION
**Pull request by florisvdg**

### CODE REVIEW

File: src/commands/install.yml [hunk 1]
```diff
@@ -20,15 +20,14 @@ steps:
...
         # If the version is not explicitly specified then use the existing one if possible.
-        if [ "<< parameters.version >>" = "latest" ]  && command -v secrethub >/dev/null 2>&1; then
+        if [ "<< parameters.version >>" = "latest" ] && command -v secrethub >/dev/null 2>&1; then
             exit 0
         fi
 
         # Get the latest version.
         VERSION="<< parameters.version >>"
         if [ "$VERSION" = "latest" ]; then
-            VERSION=$(curl --silent --show-error --location --fail --retry 3 \
```
Comment (SimonBarendse) [inline src/commands/install.yml:30]: Where did all these options go? `--show-error` `--retry 3` etc. ?

Comment (SimonBarendse) [inline src/commands/install.yml:30]: What's `$4` in this case?

Comment (florisvdg) [reply src/commands/install.yml:30]: It's parsing the json: `"tag_name": "v0.35.0"`

Comment (florisvdg) [reply src/commands/install.yml:30]: They're there: `-sSfL`. Only the `retry 3` is not there, do we want that?

Comment (florisvdg) [reply src/commands/install.yml:30]: This is the one-liner we use in `get.secrethub.io` as well.

Comment (SimonBarendse) [reply src/commands/install.yml:30]: I don't see a reason to change it now.

**Review by SimonBarendse — APPROVED** [review]

**Pull request merged**